x-pack/plugins/observability/public/pages/alerts/render_cell_value.test.tsx

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
This is incomplete, I only added as much as I needed to test my changes. Expanding on this should be easy :)